City-of-Helsinki/drupal-hdbt
# UHF-1756 data attributes

Testing instructions: 
- `make up`
- `composer require drupal/hdbt:dev-UHF-1756_data_attributes`
- `make drush-cr`
- Go to https://helfi.docker.so/en/resident-parking-permit-renewal
   - Test that the desktop menu toggle work as before (hover and click to expand menu items) 
   - Test that the mobile menu works as before, check also "scroll up" functionality on language selector links
